lib/android/audio.nit

```diff
@@ -377,6 +377,11 @@ class MediaPlayer
 
 	# Load a sound for a given resource id
 	fun load_sound(id: Int, context: NativeActivity): Music do
+		if playing then 
+			stop
+			reset
+			destroy
```

Comment:
Because if you don't want to use the same music, but actually change the music playing now, you need to release the precedent music before loading the new one. Bad side of this is that if you reload the same resource, you release it then reload it, but it only happens when pausing and resuming the app, and specifically in friendz because the `Game` class is reinstanciated every time the app is resumed.
Note that the `load_sound` method use the android factory to obtain a new `NativeMediaPlayer` every time it loads a Music, because it's the easiest way to do that.